project/src/main/java/com/google/sps/data/Candidate.java

```diff
@@ -14,4 +14,31 @@
 
 package com.google.sps.data;
 
-public class Candidate {}
+public class Candidate extends DirectoryCandidate {
+  private String email;
+  private String phoneNumber;
+  private String photoURL;
+  private String position;
+  private String website;
+  private String twitter;
+
+  public Candidate(
+      String id,
+      String name,
+      String partyAffiliation,
+      boolean isIncumbent,
```

Comment:
nit. Group all the string parameters together and include the boolean one at the end. Also reformat this so that not every parameter is in a new line. I think you should be able to fit at least four parameters in one line without exceeding the 100 character limit?